api/v2/checluster_types.go

```diff
@@ -541,6 +546,54 @@ type PodSecurityContext struct {
 	FsGroup *int64 `json:"fsGroup,omitempty"`
 }
 
+type CheClusterGitServices struct {
+	// +optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="GitHub"
+	GitHub []GitHubService `json:"github,omitempty"`
+	// +optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="GitLab"
+	GitLab []GitLabService `json:"gitlab,omitempty"`
+	// +optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="BitBucket"
```

Comment:
Bitbucket is not camel case, it's one word (https://bitbucket.org/). It's up to you to update the `struct` field or not but I think that updating the `displayName` is important.
```suggestion
	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Bitbucket"
```